llmproxy.config.yml

```diff
@@ -67,3 +67,12 @@ provider_settings:
       - code-bison
       - codechat-bison
       - code-gecko
+
+  - provider: Anthropic
+    api_key_var: ANTHROPIC_API_KEY
+    temperature: 0.1
+    max_output_tokens: 256
+    models:
+      - claude-3-opus-20240229
```

Comment:
Is it necessary to add the number at the end of the model name? Or will the latest version of that model be automatically used. If it is automatically used, it's best to keep the numbers out; If not then ignore this

Reply:
Yes, you need to specify the release date. 

claude-3-opus-20240229 
Note: 20240229 is the release date.